### ISSUE TITLE
environment variable setup

### ISSUE THREAD

**Issue by PanupongDeve** [elysiajs/documentation#452]
### What is the type of issue?

Documentation is missing

### What is the issue?

Hi, I skimmed through the ElysiaJS documentation but didn’t find information on how to set up environment variables in ElysiaJS. I think we should include this topic as a guideline for new developers.

**Comment by Crenshinibon** [comment]
Hey. I had the same question. And actually it's more of a Bun thing ...

https://bun.sh/guides/runtime/read-env?_gl=1*1857327*_ga*MTg0MjIyMTIzLjE3NTgwMDE4NDY.*_ga_3683R8TZ5M*czE3NTgwMzA1NzYkbzQkZzEkdDE3NTgwMzA1NzckajU5JGwwJGgxMTEwMzE2NzM2

A hint in the elysia docs would be nice, though.

**Comment by Crenshinibon** [comment]
Yes you are right, it's not necessarily an Elysia thing. But it's so common, that you need to access an env var in some way. And other frameworks like Sveltekit (https://svelte.dev/docs/kit/$env-dynamic-private), for example, provide a way to do so. So even if Elysia has nothing directly todo with that, you could add a short paragraph to the docs, showing a common way, for reading env-var values.

**Comment by fecony** [comment]
> other frameworks like Sveltekit (https://svelte.dev/docs/kit/$env-dynamic-private), for example, provide a way to do so. 

They do so because they apply some magic to give you access to it through their `$env/` import
Still if we add something like that, then we might need to teach people more about Web Standards, REST API best practices and so on, that would be out of scope of the framework, there is magic inside, but not for basics
